client/package.json

```diff
@@ -70,6 +70,7 @@
     "@types/socket.io-client": "^1.4.32",
     "@types/video.js": "^7.3.3",
     "@types/webtorrent": "^0.107.0",
+    "angular-resize-event": "^1.2.1",
```

Comment:
Could we avoid this extra dependency by listening the window resize event? (we already do this in https://github.com/Chocobozzz/PeerTube/blob/develop/client/src/app/app.component.ts#L94)

Reply:
I realise now that our layout prevents the left menu from changing the main container width, so it should be fine since no resizing beyond the window resize is likely to occur.